packages/smooth_app/lib/pages/personalized_ranking_page.dart

```diff
@@ -54,6 +54,8 @@ class _PersonalizedRankingPageState extends State<PersonalizedRankingPage> {
         context.watch<ProductPreferences>();
     final LocalDatabase localDatabase = context.watch<LocalDatabase>();
     final DaoProductList daoProductList = DaoProductList(localDatabase);
+    final ThemeData themeData = Theme.of(context);
+    final ColorScheme colorScheme = Theme.of(context).colorScheme;
```

Comment:
Since you already have the value of `Theme.of(context)` one line beforr, please re-use it instead of requesting it once again

Reply:
hi @g123k thank you for the comment.Now,I have refactor the code.